#Frames field 
uint32 FrameIndex
float64 FrameTime
string Sensor
Obstacle[] obstacles_array
